LayoutTests/fast/attachment/attachment-title-with-rtl.html

```diff
@@ -1,5 +1,8 @@
 <!DOCTYPE html><!-- webkit-test-runner [ AttachmentElementEnabled=true ] -->
 <html>
+<head>
+<meta name="fuzzy" content="maxDifference=0-100; totalPixels=0-50" />
```

Comment:
Hmm, why is this needed? Does the text layout differ on some platforms with these control characters inserted?

Reply:
Here's an example of error I got: https://ews-build.s3-us-west-2.amazonaws.com/macOS-Monterey-Release-WK2-Tests-EWS/6f82ec3e-2814/results.html (expand the first failure). You'll notice some very slightly different pixels around a couple of characters in the 2nd attachment.
I believe the only difference I introduced are those two zero-width spaces around the dot, which somehow must disturb the exact sub-pixel location near the start of the string. Should I file an issue for that?

Reply:
As discussed, I'll wait until we have some understanding of what's happening, before landing this patch (in case there's a more appropriate fix for the test).

Reply:
I've tweaked the test to avoid this issue (while still testing RTL characters).